### PR TITLE
Add printThis via single package.json w/ git auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10724 
@cdnjs/intern3 please help me review this PR, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/jasonday/printThis
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 47
   - Count of stars: 290
   - Count of forks: 198

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
@hare1039 I revised the commit msg, please help me review this again, thanks!

**Comment by clairetsai818** [comment]
@hare1039 sorry that I didn't notice that, please help me review this again, thanks!

File: ajax/libs/printThis/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "name": "printThis",
+  "description": "Printing plug-in for jQuery",
+  "filename": "printThis.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jasonday/printThis.git"
+  },
+  "keywords": [
+    "print",
+    "jQuery"
```
Comment (extend1994) [inline ajax/libs/printThis/package.json]: I think you can add some keywords for the author to help users find the library more easily :+1:

File: ajax/libs/printThis/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "author": "Jason Day",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jasonday/printThis.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "printThis*.(js|css|map)"
```
Comment (extend1994) [inline ajax/libs/printThis/package.json]: printThis*.+(js|css|map)

**Comment by clairetsai818** [comment]
I added some keywords and revised the file map.

File: ajax/libs/printThis/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "Printing plug-in for jQuery",
+  "filename": "printThis.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jasonday/printThis.git"
+  },
+  "keywords": [
+    "print",
+    "jQuery",
+    "page title",
```
Comment (extend1994) [inline ajax/libs/printThis/package.json]: We prefer not to separated words in `keywords` exception the condition it needs more than one word to represent the meaning. Just suggestion for you: DOM, styling, plugin can be added. You can confirm if they are good keywords by googling them, the sequence you input also has an impact on the search result. For example, we will see different result after inputing `Print DOM styling jQuery plugin` and `Print jQuery DOM styling  plugin`

**Comment by clairetsai818** [comment]
@extend1994 Thanks for your help. added the keywords, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@hare1039 How's your part?

**Comment by hare1039** [comment]
@pvnr0082t Great! Let's proceed!

**Comment by PeterDaveHello** [comment]
need a rebase ...

**Comment by PeterDaveHello** [comment]
@cdnjs/intern3 next time if you leave the PR review for more than 1 day, please help rebase it.

**Comment by clairetsai818** [comment]
@PeterDaveHello please help me review this, thanks!

File: ajax/libs/printThis/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "keywords": [
+    "print",
+    "jQuery",
+    "DOM",
+    "styling",
+    "plugin"
+  ],
+  "author": "Jason Day",
+  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/printThis/package.json]: confirmed license?
https://github.com/jasonday/printThis/blob/v1.9.0/printThis.jquery.json#L13

Comment (clairetsai818) [inline ajax/libs/printThis/package.json]: The author has modified the license info in `printThis.jquery.json`

Comment (PeterDaveHello) [inline ajax/libs/printThis/package.json]: You should give me reference when you say that.

Comment (clairetsai818) [inline ajax/libs/printThis/package.json]: ref: https://github.com/jasonday/printThis/blob/master/printThis.jquery.json

Comment (PeterDaveHello) [inline ajax/libs/printThis/package.json]: Please use precise reference next time, e.g. https://github.com/jasonday/printThis/commit/281933e513738f71bb7ddad6f22247e98d486a87, master branch changes anytime.

**Comment by clairetsai818** [comment]
@PeterDaveHello I revised the commit msg, please help me review this, thanks!

**Pull request merged**